src/card/CardImage.tsx

```diff
@@ -10,7 +10,7 @@ const CardImage: RneFunctionComponent<ImageProps> = ({ style, ...props }) => {
 
 const styles = StyleSheet.create({
   image: {
-    width: null,
+    width: undefined,
```

Comment:
This is changing the behavior of the component. Either we should turn on ignoring null checks or find a different solution.

Reply:
Okay would gonna do that!